src/v2/Apps/Artists/Components/ArtistsArtistCard.tsx

```diff
@@ -1,37 +1,31 @@
 import React from "react"
-import { Box, BoxProps, Image, ResponsiveBox, Text } from "@artsy/palette"
+import { Box, Flex, Image, ResponsiveBox, Text } from "@artsy/palette"
 import { RouterLink } from "v2/Artsy/Router/RouterLink"
-import { FollowArtistButtonFragmentContainer } from "v2/Components/FollowButton/FollowArtistButton"
 import { ContextModule } from "@artsy/cohesion"
 import { createFragmentContainer, graphql } from "react-relay"
 import { ArtistsArtistCard_artist } from "v2/__generated__/ArtistsArtistCard_artist.graphql"
+import { FollowArtistButtonFragmentContainer } from "v2/Components/FollowButton/FollowArtistButton"
 
-interface ArtistsArtistCardProps extends BoxProps {
+interface ArtistsArtistCardProps {
   artist: ArtistsArtistCard_artist
 }
 
 export const ArtistsArtistCard: React.FC<ArtistsArtistCardProps> = ({
   artist,
-  ...rest
 }) => {
   const { image, counts } = artist
 
   if (!image) return null
 
   return (
-    <Box
-      display="flex"
-      flexDirection="column"
-      justifyContent="space-between"
-      height="100%"
-      p={1}
-      border="1px solid"
-      borderColor="black10"
-      borderRadius={2}
-      flex="1"
-      {...rest}
-    >
-      <RouterLink to={artist.href} noUnderline style={{ display: "block" }}>
+    <>
+      <RouterLink
+        to={artist.href}
+        noUnderline
+        style={{ display: "block" }}
+        // Avoid the redundant tab-stop
+        tabIndex={-1}
```

Comment:
What do you mean? Can you paste in an example? (wondering if i should be doing this too)

Reply:
This is just because in this case we've got the follow button in there so we can have one big link; so it's broken into two links and I'm just skipping the first one when tabbing via keyboard.